website/content/docs/internals/telemetry.mdx

```diff
@@ -291,6 +291,9 @@ These metrics relate to [Vault Enterprise Replication](/vault/docs/enterprise/re
 | `vault.replication.rpc.standby.server.register_lease_request` | Duration of time taken by standby register lease request                                                                                   | ms              | summary |
 | `vault.replication.rpc.standby.server.wrap_token_request`     | Duration of time taken by standby wrap token request                                                                                       | ms              | summary |
 
```

Comment:
I believe this blank line will break the Markdown formatting

Reply:
Good call Max. I copy-pasted verbatim from the original PR but I suspect it was never checked there either.

Reply:
Yep

<img width="792" alt="image" src="https://github.com/hashicorp/vault/assets/120915/4681d32b-0c5e-4ccc-8aad-d0fe621c4bb5">